src/modules/main/types/category.ts

```diff
@@ -0,0 +1,4 @@
+export interface ICategory {
```

Comment:
we probably don't need an interface for that..
what's the example of ICategory?